nlp_primitives/utilities.py

```diff
@@ -7,25 +6,25 @@
 
 
 def clean_tokens(textstr):
+    textstr = textstr.translate(str.maketrans('', '', string.punctuation))
+    textstr = [ch for ch in textstr.split(' ') if len(ch) > 0]
+
     try:
         swords = set(nltk.corpus.stopwords.words('english'))
+        to_remove = set(string.punctuation).union(swords)
+        textstr = [ch.lower() for ch in textstr if ch not in to_remove]
     except LookupError:
         nltk.download('stopwords')
         swords = set(nltk.corpus.stopwords.words('english'))
+        to_remove = set(string.punctuation).union(swords)
```

Comment:
might it be worth not repeating this and the following line twice as they are the same as on 14 and 15, and could instead be taken out of the `try`/`except` and just put after the `except` expression and not be indented? / could be put in a `finally` expression

Reply:
There's a [weird bug ]( https://github.com/FeatureLabs/evalml/issues/1083)going on in the `evalml` code right now that actually stems from this. If the internet connection is bad or non-existent when running this code, the nltk download will fail and any attempts to use that variable will throw an error. By keeping all references to nltk's stopwords within the try/except block, the code will run and successfully return something even if the download(s) fail.

Reply:
Hm that is strange, I guess there isn't a lot of python handling around downloading and waiting for stuff to download/handling if it fails in the evalml library/here. in that case, this seems very reasonable!!